api/v1alpha3/openstackmachine_types.go

```diff
@@ -57,6 +57,10 @@ type OpenStackMachineSpec struct {
 	// A networks object. Required parameter when there are multiple networks defined for the tenant.
 	// When you do not specify the networks parameter, the server attaches to the only network created for the current tenant.
 	Networks []NetworkParam `json:"networks,omitempty"`
+
+	// IP address of a port from this subnet will be marked as AccessIPv4 on the created compute instance
```

Comment:
I wonder what happens if networks and subnet which is not part of networks are given.
If we do not accept the case comment it here. Or add the check logic.

Reply:
I guess below discussion covered this? maybe need some comments update?

    if AccessSubnet is set we will get a hit here and have a value for accessIPv4
    otherwise accessIPv4 and we have no change to before this PR

Reply:
Ok, got it.

Reply:
I added f5ff1bf7272b5a7e34eac665dd7e8e5182d4c86a.
The instance creation will fail if no ports with fixed IP are found in the specified subnet. This also covers the case when this subnet does not exist or is not in `.Networks`.
I didn't notice the ...
> Ok, got it.

... until now, so please let me know if you'd like to keep this change. I can drop this commit if not needed.

Reply:
@jsen- Thanks for adding checking logic which is better to be added. Keep the commit, please.